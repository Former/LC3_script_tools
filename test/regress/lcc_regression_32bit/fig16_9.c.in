345231
46
8
-23333
-23333
7
7
-43333
52
3

