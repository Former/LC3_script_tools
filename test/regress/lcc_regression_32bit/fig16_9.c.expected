Enter 10 numbers.
Input number 0 : 345231Input number 1 : 
46
Input number 2 : 8
Input number 3 : -23333Input number 4 : 
-23333Input number 5 : 
7
Input number 6 : 7
Input number 7 : -43333Input number 8 : 
52
Input number 9 : 3
Original number 345231.  Number of repeats 1
Original number 46.  Number of repeats 1
Original number 8.  Number of repeats 1
Original number -23333.  Number of repeats 2
Original number -23333.  Number of repeats 2
Original number 7.  Number of repeats 2
Original number 7.  Number of repeats 2
Original number -43333.  Number of repeats 1
Original number 52.  Number of repeats 1
Original number 3.  Number of repeats 1
